### ISSUE TITLE
Please APA style

### ISSUE THREAD

**Issue by sammerk** [ropensci/RefManageR#74]
Please add the APA-style - it is used by a lot social scientists ...

**Comment by dmi3kno** [comment]
I wonder if it possible to wrap [one of the CSL processors](https://citationstyles.org/developers/) from other languages. Perhaps `citeproc.js` with V8?

**Comment by embruna** [comment]
Couldn't agree more. This is probably the best package around; it's by far the easiest way for including multiple reference lists in a document (CV, book). IMO the biggest shortcoming is lack of alt reference styles.

**Comment by ReneKaradakic** [comment]
I totally agree. For using xaringan this package seems to be most suited and the only thing that is really missing is the application of an appropriate citation style for my field.
